languagetool-http-client/src/test/java/org/languagetool/remote/RemoteServerTest.java

```diff
@@ -0,0 +1,15 @@
+package org.languagetool.remote;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RemoteServerTest {
+
+  @Test
+  public void toStringOutputNotNull() {
+    final RemoteServer objectUnderTest = new RemoteServer("Languagetool", "4.5-SNAPSHOT", "2019-02-05 17:54");
+    final String retval = objectUnderTest.toString();
+    Assert.assertEquals("Languagetool/4.5-SNAPSHOT/2019-02-05 17:54", retval);
```

Comment:
Is this considerable to one line? 
Assert.assertEquals("Languagetool/4.5-SNAPSHOT/2019-02-05 17:54", new RemoteServer("Languagetool", "4.5-SNAPSHOT", "2019-02-05 17:54").toString()); 

Or is that possibly too long a line length? 

In which case maybe we could drop initialising objectunderTest and create retval directly with the remoteServer object?

Reply:
Drop `retval`

Reply:
I dropped retval but didn't condense further to avoid a very long line.
Let me know if anyone thinks I should condense further.